cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_buy_button/nft_buy_button.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "/styles/variables" as v;
+
+.container {
+    align-self: flex-start;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+
+    width: fit-content;
+    background-color: #eee2e5;
+    padding-left: 8px;
+    padding-right: 8px;
+    border-radius: 5px;
+}
+
+.price {
+    padding-left: 4px;
+    padding-right: 4px;
+    margin-top: -4px;
+    margin-bottom: -4px;
+    font-size: 30px;
+    font-weight: bold;
+    height: min-content;
+    color: v.$text-color;
+}
+
+.opensea {
```

Comment:
use flex: 1 1 auto. The container is a display flex, but its children are not flex, so it can't work right now.